src/measure_theory/lp_space.lean

```diff
@@ -325,9 +325,55 @@ lemma snorm_congr_norm_ae {f : α → F} {g : α → G} (hfg : ∀ᵐ x ∂μ, 
 le_antisymm (snorm_mono_ae $ filter.eventually_eq.le hfg)
   (snorm_mono_ae $ (filter.eventually_eq.symm hfg).le)
 
+@[simp] lemma snorm'_norm {f : α → G} : snorm' (λ a, ∥f a∥) q μ = snorm' f q μ :=
```

Comment:
```suggestion
@[simp] lemma snorm'_norm {f : α → F} : snorm' (λ a, ∥f a∥) q μ = snorm' f q μ :=
```
All the other lemmas around have functions taking values in `F`, is there a reason to change for this one?